nym-vpn-core/crates/nym-vpnd/src/main.rs

```diff
@@ -264,12 +268,39 @@ fn init_sentry() -> Option<ClientInitGuard> {
             traces_sample_rate: 1.0,
             enable_logs: true,
             shutdown_timeout: Duration::from_secs(2),
+            server_name: Some(std::borrow::Cow::Borrowed("nym")),
             ..Default::default()
         },
     ));
+    sentry::configure_scope(|scope| {
+        scope.set_tag("os_version", &os_info.os_version);
+        scope.set_tag("gpu", &os_info.arch);
```

Comment:
I think knowing the gpu for vpnd is useless, better to remove it

Reply:
Also, the os_version and architecture are already set by sentry, no need to create custom tags for them, let's avoid data duplication